servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -38,8 +38,7 @@
 
 job_include_keys = attempt_include_keys + (
     'calls', 'description', 'executionEvents', 'labels', 'parentWorkflowId',
-    'returnCode', 'shardIndex', 'start', 'status', 'submission',
-    'subWorkflowId', 'workflowName')
+    'returnCode', 'status', 'submission', 'subWorkflowId', 'workflowName')
```

Comment:
this is just removing the `shardIndex` and `start` keys that were duplicated?

Reply:
yup